resources/core/charts/ui-api/scripts/filter.lua

```diff
@@ -0,0 +1,218 @@
+-- Called on the request path.
+function envoy_on_request(request_handle)
+  request_handle:logInfo("Hello World")
+  GraphQlAuthPlugin_OnRequest(request_handle)
+end
+
+-- Called on the response path.
+function envoy_on_response(response_handle)
+end
+
+function GraphQlAuthPlugin_OnRequest(request_handle)
+  -- it does not work - i'm not able to read host header
+  -- local authority = request_handle:headers():get(":authority")
+  -- if (authority == "core-ui-api.kyma-system.svc.cluster.local") then
```

Comment:
I can't see how this comment is relevant. Please remove it or convince me of its importance ;)

Reply:
Ok, will be removed.